### DISCUSSION TITLE
Tailwind Elements Stable v1.0.0. - a free, open-source UI Kit with 500+ components integrated with Next - is out.

### DISCUSSION THREAD

**Discussion by MaciejZimoch** [vercel/community#4380] [Show & Tell]
Tailwind Elements Stable v1.0.0. - a free, open-source UI Kit with 500+ components integrated with Next - is out.

And we couldn't be prouder, so give it a go [here](https://tailwind-elements.com/docs/standard/integrations/next-integration/)

It will change Your life (or at least how smooth & painless web development is) ;)

It's stable & reliable, comes with over 500 UI Components, easy theming & customisation, dark mode, Figma design system AND free hosting :) .

A popular React framework, Next has never had such ease of development and versatility as it does with Tailwind Elements.

For all JS aficionados: By default all components have autoinit which means they are initialised by data attributes. But if you want to make init by JS - there is also possibility to do that.

Check out how to do it and more about Tailwind Elements [here](https://tailwind-elements.com/docs/standard/integrations/next-integration/)

Even though TE is already a fully functional UI Kit, it's still in its infancy really - we have great plans and a long and exciting road ahead of us. So whatever input (however critical) You might have, please share it with us here or on any of our channels on which we will be more than happy to host you:
[Discord](https://discord.com/invite/ZkTVKaJC) 
[GitHub](https://github.com/mdbootstrap/Tailwind-Elements) 
[X ](https://twitter.com/TailwindElement)

**Comment by alp82** [comment]
Had a quick look at the docs and I don't like the fact that we have to use `useEffect` to integrate tailwinds elements components. I try to avoid using side effects (a.k.a. life-cycle hooks) as much as possible.

I prefer the install-free approach of https://ui.shadcn.com/ which makes the whole process dependency-free, lightweight and highly customizable.

Also a big fan of https://tailwindui.com/ which is similar to the above, but without the handy `npx` commands.
